graph/graph.go

```diff
@@ -424,11 +462,10 @@ func (graph *Graph) Release(sessionID string, layerIDs ...string) {
 // A head is an image which is not the parent of another image in the graph.
 func (graph *Graph) Heads() map[string]*image.Image {
 	heads := make(map[string]*image.Image)
-	byParent := graph.ByParent()
 	graph.walkAll(func(image *image.Image) {
 		// If it's not in the byParent lookup table, then
```

Comment:
Maybe `If it's not in the byParent lookup table` this comment should be removed? since there is 
no `byParent` lookup table  :)